test/let_spec.rb

```diff
@@ -0,0 +1,45 @@
+BDDSM.describe 'BDDSM::Describe.let' do # rubocop:disable Metrics/BlockLength
+  # let should define variable accessible from 'it'
+  let(:a) { 1 }
+  it { expect(a).to eq 1 }
+
+  # let variable should be accessible from another let
+  let(:one) { 1 }
+  let(:two) { one + one }
+  it { expect(two).to eq 2 }
+
+  # let variable should be overriden
+  let(:three) { 3 }
+  let(:three) { 4 }
+  it { expect(three).to eq 4 }
+
+  # checks if let block executes only once
+  module Count # rubocop:disable Lint/ConstantDefinitionInBlock
+    module_function
+
+    @instance_count ||= 0
+
+    def instance_count
+      @instance_count
+    end
+
+    def test(counter)
+      @instance_count += 1
+      counter + 1
+    end
+  end
+
+  # test module class variables works
+  it do
```

Comment:
This test is unclear. It is impossible to understand, why expect(Count.instance_count).to eq 2

Reply:
with instances it seems more clear?

Reply:
not really

Reply:
@AlexNimoy @sas2job @sk8higher could you suggest something in here?

Reply:
have no ideas

Reply:
have no ideas either

Reply:
If is is impossible to use instance variable, let's try to use class variables with `@@calls_count`

Reply:
ok, i used this snippet but inside class  to share class variable between expect instances